library/src/main/java/com/cube/storm/ui/fragment/StormTabbedFragment.java

```diff
@@ -36,17 +35,11 @@
  */
 public abstract class StormTabbedFragment extends Fragment implements StormInterface, ViewPager.OnPageChangeListener
 {
-	/**
-	 * The Storm descriptor src of the tab that should be selected on startup
-	 */
-	private static final String EXTRA_START_TAB_SRC = "extra_start_tab_src";
```

Comment:
What was the reason to remove this functionality? I would need to check, but it is a breaking change and I think some apps will use this to control the initially displayed tab.

Reply:
As we talked, this static string is not used anymore